cogs/DetectAI.py

```diff
@@ -22,41 +281,49 @@ def __init__(self, bot: commands.Bot) -> None:
         description="The text to run through the AI.",
         required=True,
     )
-    @option(
-        "ephemeral",
-        bool,
-        description="Whether the response should be ephemeral or not. Only staff can make this False.",
-        required=False,
-        default=True,
-    )
```

Comment:
Re-add staff only functionality of the command as discussed here:
https://discord.com/channels/238956364729155585/1040360228388098159/1099443748871610470